### DISCUSSION TITLE
Payroll Engine Setup & Consultation

### DISCUSSION THREAD

**Discussion by Abdullahifisol** [Payroll-Engine/PayrollEngine#5] [General]
Hi Jani, 

Can you provide paid consultation? We need some guidance and assistance in setting up the backend code, which we are currently having trouble with.

please let me know.

**Comment by Giannoudis** [comment]
I am glad to support you with your problems with the backend. Please send me an email via [PayrollEngine.org](https://payrollengine.org/).
